cmd/confluent/main.go

```diff
@@ -38,6 +40,9 @@ func main() {
 	cmd := pcmd.NewConfluentCommand(cfg)
 
 	if err := pcmd.Execute(cmd, os.Args[1:], cfg); err != nil {
+		if _, ok := err.(*ppanic.Panic); ok {
+			output.ErrPrintln(err.Error())
```

Comment:
will traces info be printed out still?

Reply:
No, only a message that is formatted like the other errors in the CLI (i.e. "Error: ..."). Maybe we should make a note as well that a panic occurred and was sent to cli-service?

Reply:
I think we'll have to add "Error: " here manually, right?

Reply:
Might want to call https://pkg.go.dev/github.com/spf13/cobra#CheckErr